kitsune/wiki/jinja2/wiki/document.html

```diff
@@ -56,7 +56,9 @@
     {{ search_box(settings, id='support-search-sidebar', params=search_params) }}
   </div>
 
-  {{ aaq_widget(request) }}
+  {% if default_slug != 'get-community-support' %}
```

Comment:
I would probably move the check inside the view and expose a variable like `show_aaq_widget` in the context. Up to your discretion if you would like to go down this path.